Ds3/Helpers/Strategies/StreamFactory/ReadStreamStreamFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+namespace Ds3.Helpers.Strategies.StreamFactory
+{
+    /// <summary>
+    /// Create one stream and GET blobs in order without seeking
+    /// </summary>
+    public class ReadStreamStreamFactory : StreamFactory
```

Comment:
Normally, I'd wrinkle my nose at an empty class, but I think it makes sense in this case.